2017/11.md

```diff
@@ -34,8 +34,12 @@ TBD Hotels nearby (optional)
 
 ## Agenda Topic Rules
 
-1. Proposals looking to advance must be added to the agenda along with necessary review materials 7 days prior to the first day of the meeting.
-   1. Note: any items added after 7 days prior to the start time of the meeting must be denoted with an ️:hourglass: symbol, which indicates that it is not eligible for stage advancement, and that it must wait until all other items have been addressed.
+Deadline for advancement eligibility: **November 18th, 2017, 18:00 UTC**
```

Comment:
Link to https://www.timeanddate.com/worldclock/fixedtime.html?msg=TC39+Submission+deadline&iso=20171118T18&p1=1440 or https://www.timeanddate.com/countdown/generic?p0=1440&iso=20171118T18&msg=TC39%20Submission%20deadline ?

Reply:
Nice idea, I like the countdown